tests/connectors/document_index/test_document_index.py

```diff
@@ -200,6 +201,27 @@ def test_document_index_searches_asymmetrically(
     assert "Mark" in search_result[0].section
 
 
+def test_document_index_hybrid_search_combines_semantic_and_keyword_search(
+    document_index: DocumentIndexClient, collection_path: CollectionPath
+) -> None:
+    document_path = DocumentPath(
+        collection_path=collection_path,
+        document_name="test_document_index_hybrid_search_combines_semantic_and_keyword_search",  # is always there
+    )
+    search_query = SearchQuery(
+        query="Which food do humans like?",
+        max_results=10,
+        min_score=0.0,
```

Comment:
Maybe not one for this PR, but we want to discourage use of `min_score` with searches over hybrid indexes (the re-ranked score is not very useful...), and a min_score of 0.0 does nothing anyway. We should probably make it an optional parameter that defaults to 0.

Reply:
(I'll raise a task for this)

Reply:
(PR for this: https://github.com/Aleph-Alpha/intelligence-layer-sdk/pull/1053)

Reply:
Should we then add this into the docstring or give a warning for this? Im not quite sure of the implications and would let you decide

Reply:
There are two things here:
- min score of 0.0 does nothing: this is the case already and I don't think this needs documenting; given a score is documented as being between 0 and 1, it is intuitive that filtering by >= 0 will always yield a full result set.
- score is not very useful for hybrid indices: this should now be sufficiently documented in the DocumentIndexRetriever once https://github.com/Aleph-Alpha/intelligence-layer-sdk/pull/1053 is merged. We also document this in the DI OpenAPI spec, so I see no need to modify the IL DI client as well.